### ISSUE TITLE
Flying Golems Trampling Crops

### ISSUE THREAD

**Issue by majikguy** [Azanor/thaumcraft-5#451]
Not really sure if it should be considered a bug, but flying golems still trample and break crops. Tried to set up a terraced farm with flying farmers, but they quickly ground the farm into dirt.
Not sure how hard it would be to change this, or if you would want to, but it seems really off to have them trample tilled soil without actually touching it.

**Comment by Azanor** [comment]
What material were they made from? 'Light' golems and flyers should not trample crops, but heavy golem, even if flyers, will. If not, then there might be a bug.

**Comment by majikguy** [comment]
It was a Brass flyer with the carrying harness, biothaumic head, and dexterous hands, so I am pretty sure it should still have a light frame? Not near my PC at the moment so I can't double check. 
Cool to see that you had thought of that, regardless of whether or not it is completely working. :-)

**Comment by Azanor** [comment]
Turns out that at some point things have changed in the background. The canTriggerWalking method is not being used by farmland to check if it should become dirt anymore. :( 
Still, not that hard to fix.

**Comment by Azanor** [comment]
Actually, nevermind - turns out it is not that simple to fix. This should be fixed in vanilla MC code.

**Comment by Azanor** [comment]
https://github.com/MinecraftForge/MinecraftForge/issues/2510

**Comment by majikguy** [comment]
Yeah, I figured that it would require a change outside your jurisdiction.
Been looking for something to work on, I may see if I can dig around in the
Forge code and look for a fix for this.
Thanks for your time. :)
